### DISCUSSION TITLE
firedancer single node stuck at slot 252

### DISCUSSION THREAD

**Discussion by mingdourolabs** [firedancer-io/firedancer#3236] [Support]
Hi, I'm trying to run a single node firedancer for testing.
I got v0.115.20013 and built, then I created genesis using solana-genesis tool (since I couldn't find the same tool in firedancer code base).
`SOL="000000000"

~/git/solana/target/release/solana-genesis \
	--bootstrap-validator-lamports 100$SOL \
	--bootstrap-validator-stake-lamports 100$SOL \
	--bootstrap-validator /home/ubuntu/keys/identity.json /home/ubuntu/keys/vote-account.json /home/ubuntu/keys/stake-account.json \
	--enable-warmup-epochs \
	--faucet-pubkey /home/ubuntu/keys/faucet.json \
	--faucet-lamports 30000$SOL \
	--fee-burn-percentage 100 \
	--inflation none \
	--ledger /home/ubuntu/ledger \
	--rent-burn-percentage 100 \
	--vote-commission-percentage 100`

Then I ran the following command
`sudo build/native/gcc/bin/fddev run  --config ~/config/single/single.toml`

single.toml content
`user = "ubuntu"
scratch_directory = "/home/{user}"

[development]
bootstrap=true
sandbox=false

[ledger]
path="/home/{user}/ledger"

[rpc]
port = 8899
full_api = true
only_known = false
private = true

[consensus]
vote_account_path="/home/{user}/keys/vote-account.json"
identity_path="/home/{user}/keys/identity.json"
wait_for_vote_to_start_leader = false

[snapshots]
full_snapshot_interval_slots = 1200

[log]
path="/home/{user}/logs/single.log"
level_logfile = "DEBUG"`

It started ok, and then got stuck at slot 252, root slot 221. This is reproducible every time.
I tried running solana-validator from solana repo build with the same genesis, it seems to work.

I tried using different versions of firedancer and didn't get any luck.
with monitor tool, I can see everything fine up till the point of issue, then it shows poh is 99% backp cnt, "in backp" err.

Any idea?

**Comment by ripatel-fd** [comment]
This has been fixed in the latest release.
